app/src/debug/java/org/simple/clinic/di/DebugCrashReporterModule.kt

```diff
@@ -0,0 +1,15 @@
+package org.simple.clinic.di
+
+import org.simple.clinic.crash.CrashReporterModule
+import org.simple.clinic.crash.NoOpCrashReporter
+import org.simple.clinic.facility.FacilityRepository
+import org.simple.clinic.user.UserSession
+
+class DebugCrashReporterModule : CrashReporterModule() {
```

Comment:
We should stop doing this. Please see section **Option 1: Override bindings by subclassing modules (don’t do this!)** at https://dagger.dev/testing

Reply:
Agreed. I have raised one here: https://www.pivotaltracker.com/story/show/167975791

Let's pick this up in the coming iteration.